### PR TITLE
[eas-cli] detect missing android or ios directories

### LINKED ISSUE #114: [eas build] Managed project with workflow set to generic in eas.json outputs incorrect error message
In a managed project (no ios or android folder), running the following produces an error message that doesn't seem right.
```
client 𝝠 easd build:create -p ios
✔ Ensuring @bacon/pillar-valley is created on Expo

We've found multiple schemes in your Xcode project: 
You can specify the scheme you want to build at builds.ios.PROFILE_NAME.scheme in eas.json.
```
I'd expect the project to not have any schemes, but here it says it found multiple.

### PR DESCRIPTION
**Pull request by wkozyra95**
# Why

Fixes #114

# How

Better error message

# Test Plan

run generic build in manged project

### CODE REVIEW

File: packages/eas-cli/src/build/ios/build.ts [hunk 1]
```diff
@@ -24,6 +26,15 @@ export async function prepareIosBuildAsync(
     easConfig,
   });
 
+  if (
+    buildCtx.buildProfile.workflow === Workflow.Generic &&
+    !(await fs.pathExists(path.join(commandCtx.projectDir, 'ios')))
+  ) {
+    throw new Error(
+      '"ios" directory not found. If you want to build managed project update builds.ios.PROFILE_NAME.workflow in "eas.json".'
```
Comment (dsokal) [inline packages/eas-cli/src/build/ios/build.ts:34]:
```suggestion
      '"ios" directory not found. If you're trying to build a managed project, set builds.ios.PROFILE_NAME.workflow in "eas.json" to "managed".'
```

Comment (dsokal) [reply packages/eas-cli/src/build/ios/build.ts:34]: Is it possible to replace `PROFILE_NAME` with the actual name?

File: packages/eas-cli/src/build/android/build.ts [hunk 2]
```diff
@@ -25,8 +27,17 @@ export async function prepareAndroidBuildAsync(
...
   });
-
   const { buildProfile } = buildCtx;
+
+  if (
+    buildProfile.workflow === Workflow.Generic &&
+    !(await fs.pathExists(path.join(commandCtx.projectDir, 'android')))
+  ) {
+    throw new Error(
+      '"android" directory not found. If you want to build managed project update builds.android.PROFILE_NAME.workflow in "eas.json".'
```
Comment (dsokal) [inline packages/eas-cli/src/build/android/build.ts:37]:
```suggestion
      '"android" directory not found. If you're trying to build a managed project, set builds.android.PROFILE_NAME.workflow in "eas.json" to "managed".'
```

Comment (dsokal) [reply packages/eas-cli/src/build/android/build.ts:37]: Is it possible to replace `PROFILE_NAME` with the actual name?

**Review by dsokal — CHANGES_REQUESTED** [review]

File: packages/eas-cli/src/build/android/build.ts (hunk 2 above)
Comment (wkozyra95) [reply packages/eas-cli/src/build/android/build.ts:37]: we use PROFILE_NAME this way for schema already

Comment (dsokal) [reply packages/eas-cli/src/build/android/build.ts:37]: I'm not sure if I know what you mean.

`builds.android.${commandCtx.profile}.workflow` should do the trick.

**Review by dsokal — APPROVED** [review]
LGTM but maybe it's worth waiting for @brentvatne / @EvanBacon to check English.

**Review by EvanBacon — APPROVED** [review]
lgtm

File: packages/eas-cli/src/build/android/build.ts [hunk 3]
```diff
@@ -25,8 +27,17 @@ export async function prepareAndroidBuildAsync(
...
   });
-
   const { buildProfile } = buildCtx;
+
+  if (
+    buildProfile.workflow === Workflow.Generic &&
+    !(await fs.pathExists(path.join(commandCtx.projectDir, 'android')))
+  ) {
+    throw new Error(
+      `"android" directory not found. If you're trying to build a managed project, set builds.android.${commandCtx.profile}.workflow in "eas.json" to "managed".`
```
Comment (EvanBacon) [inline packages/eas-cli/src/build/android/build.ts:37]:
```suggestion
      `"android/" directory not found. If you're trying to build a managed project, set builds.android.${commandCtx.profile}.workflow in "eas.json" to "managed".`
```
use some chalk.bold effect on the `build.android...`

**Review by brentvatne — APPROVED** [review]

**Pull request merged**